### PR TITLE
fix typos in comments and relay tutorial

### PR DESCRIPTION
**Pull request by jiangzoi**
@zhiics
Hi Zhi,

Please review my pull request when you have time. It is related to typo fix in code comments and tutorial.

Thank you so much!
Zheng

### CODE REVIEW

File: include/tvm/runtime/object.h [hunk 1]
```diff
@@ -1,877 +0,0 @@
-/*
```
Comment (ymwangg) [inline include/tvm/runtime/object.h:1]: Why this file is deleted?

Comment (jiangzoi) [reply include/tvm/runtime/object.h]: It is deleted by mistake. I will update the PR soon. Thank you for your help!

**Comment by zhiics** [comment]
Thanks for fixing. Could you retrigger the CI? The error should be irrelevant.

**Comment by junrushao1994** [comment]
Looks like a flaky test. Saw it several times this week.

**Comment by jiangzoi** [comment]
> flaky

Thanks Junru for help, should I open a new pull request to see whether it could be fixed?

**Comment by junrushao1994** [comment]
The flakiness is fixed this morning. Could you push an empty commit to re-trigger the CI?

```
git commit --allow-empty -m "retrigger"
git push
````

**Comment by zhiics** [comment]
Thanks @jiangzoi @ymwangg @junrushao1994

**Comment by jiangzoi** [comment]
Thank you for your help for my first PR! @zhiics @ymwangg @junrushao1994

**Pull request merged**